netket/graph/symmetry.py

```diff
@@ -136,6 +141,20 @@ def product_table(self):
 
         return product_table
 
+    def inverse(self):
+        if np.all(self._inverse) == None:
```

Comment:
why np.all ? shouldn't inverse just be either None or what it must be?

you can check with `if self._inverse is None:`

Reply:
That is a much better solution, thanks!